src/coreclr/vm/codeman.cpp

```diff
@@ -1258,29 +1258,18 @@ void EEJitManager::SetCpuInfo()
     int cpuFeatures = minipal_getcpufeatures();
 
 #if defined(TARGET_X86) || defined(TARGET_AMD64)
-
-#if defined(TARGET_X86) && !defined(TARGET_WINDOWS)
-    // Linux may still support no SSE/SSE2 for 32-bit
-    if ((cpuFeatures & XArchIntrinsicConstants_VectorT128) == 0)
-    {
-        EEPOLICY_HANDLE_FATAL_ERROR_WITH_MESSAGE(COR_E_EXECUTIONENGINE, W("SSE and SSE2 processor support required."));
```

Comment:
I can figure out how to add this back if we care enough about 32-bit Linux on ancient hardware (I do not not.)